code/game/objects/items.dm

```diff
@@ -811,3 +811,6 @@
 			step_towards(src,S)
 		else ..()
 
+//Gets the rating of the item, used in stuff like machine construction.
+/obj/item/proc/get_rating()
+	return 1
```

Comment:
Should really be 0.
The average item has no use in machine construction, and it lets you check get_rating() to see if an item is usable.